packages/prediction-app/src/components/BacktestsTable/BacktestsTable.tsx

```diff
@@ -0,0 +1,235 @@
+    COMPLETED: 'completed',
+    RUNNING: 'running',
+    FAILED: 'failed',
+}
+
+const labelByStatus = {
+    [statuses.COMPLETED]: i18n.t('Completed'),
+    [statuses.RUNNING]: i18n.t('Running'),
+    [statuses.FAILED]: i18n.t('Failed'),
+}
+
+const columnHelper = createColumnHelper<BackTestRead & { status?: string }>();
+
+const columns = [
+    columnHelper.accessor('id', {
+        header: i18n.t('ID'),
+        filterFn: 'equals',
+    }),
+    columnHelper.accessor('name', {
+        header: i18n.t('Name'),
+        filterFn: 'includesString',
+    }),
+    columnHelper.accessor('created', {
+        header: i18n.t('Created'),
+        cell: (info) => info.getValue() ? new Date(info.getValue()!).toLocaleString() : undefined,
+    }),
+    columnHelper.accessor('modelId', {
+        header: i18n.t('Model'),
+        filterFn: 'equals',
+    }),
+    columnHelper.accessor('status', {
+        header: i18n.t('Status'),
+        filterFn: 'equals',
+        enableSorting: false,
+        cell: (info) => info.getValue() ? (
+            <Chip dense>
+                {labelByStatus[info.getValue() as keyof typeof labelByStatus]}
+            </Chip>
+        ) : null,
+    }),
+    columnHelper.accessor('startDate', {
+        header: i18n.t('Start Date'),
+        cell: (info) => info.getValue() ? new Date(info.getValue()!).toLocaleString() : undefined,
+    }),
+    columnHelper.accessor('endDate', {
+        header: i18n.t('End Date'),
+        cell: (info) => info.getValue() ? new Date(info.getValue()!).toLocaleString() : undefined,
+    }),
+    columnHelper.display({
+        id: 'actions',
+        header: i18n.t('Actions'),
+        cell: (info) => (
+            <BacktestActionsMenu
+                id={info.row.original.id}
+                name={info.row.original.name}
+            />
+        ),
+    }),
+];
+
+const getSortDirection = (column: any) => {
+    if (!column.getIsSorted()) return 'default';
+    return column.getIsSorted() as 'asc' | 'desc';
+};
+
+type Props = {
+    backtests: BackTestRead[];
+    models: ModelSpecRead[];
+}
+
+export const BacktestsTable = ({ backtests, models }: Props) => {
+    const navigate = useNavigate();
+    const table = useReactTable({
+        data: backtests || [],
+        columns,
+        initialState: {
+            sorting: [{ id: 'created', desc: true }],
+            columnVisibility: {
+                startDate: false,
+                endDate: false,
+                status: false,
+            },
+        },
+        getRowId: (row) => row.id.toString(),
+        enableRowSelection: true,
+        getSortedRowModel: getSortedRowModel(),
+        getCoreRowModel: getCoreRowModel(),
+        getFilteredRowModel: getFilteredRowModel(),
+        getPaginationRowModel: getPaginationRowModel(),
+    });
+
+    const hasVisibleRows = table.getRowModel().rows.length > 0;
+
+    return (
+        <div>
+            {(table.getIsSomeRowsSelected() || table.getIsAllRowsSelected()) ? (
```

Comment:
I would think `getIsSomeRowsSelected` would be true whenever `getIsAllRowsSelected` is true?